### ISSUE TITLE
`make test-watch-for-repl` fails on `MacOS`

### ISSUE THREAD

**Issue by siddarthkay** [status-im/status-react#20041]
# Problem

As reported by @seanstrom 

`make test-watch-for-repl` fails with : 

```
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
make[1]: Entering directory '/Users/siddarthkumar/code/experiments/status-mobile/build'
  CXX(target) Release/obj.target/status_nodejs_addon/modules/react-native-status/nodejs/status.o
error: unknown target CPU 'armv8.3-a+crypto+sha2+aes+crc+fp16+lse+simd+ras+rdm+rcpc'
note: valid target CPU values are: nocona, core2, penryn, bonnell, atom, silvermont, slm, 
goldmont, goldmont-plus, tremont, nehalem, corei7, westmere, sandybridge, corei7-avx, ivybridge, 
core-avx-i, haswell, core-avx2, broadwell, skylake, skylake-avx512, skx, cascadelake, cooperlake, 
cannonlake, icelake-client, rocketlake, icelake-server, tigerlake, sapphirerapids, alderlake, raptorlake, 
meteorlake, sierraforest, grandridge, graniterapids, emeraldrapids, knl, knm, k8, athlon64, athlon-fx, 
opteron, k8-sse3, athlon64-sse3, opteron-sse3, amdfam10, barcelona, btver1, btver2, bdver1, bdver2, 
bdver3, bdver4, znver1, znver2, znver3, znver4, x86-64, x86-64-v2, x86-64-v3, x86-64-v4
```

This seems like an issue with `node-pre-gyp rebuild`

I tested this on my Linux box and it seemed to not fail this way.

**Comment by siddarthkay** [comment]
One thing I immediately noticed was that this make command is missing a target 

https://github.com/status-im/status-mobile/blob/8f0840e913506c0c32157f403e72f5ecb80768b0/Makefile#L356-L365

**Comment by siddarthkay** [comment]
Attaching target similar to other commands building `node-pre-gyp` should fix this issue.
Which would be `clojure` but `default` should also work.

I had seen similar issues with `node-pre-gyp` which were fixed by upgrading `python` to `3.1` and that could be missing when we do not specify a shell target.

Indeed default shell has `python 3.1`

https://github.com/status-im/status-mobile/blob/8f0840e913506c0c32157f403e72f5ecb80768b0/nix/shell.nix#L26
